tests/lax_numpy_test.py

```diff
@@ -288,18 +286,18 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
               tolerance={np.float16: 1e-2}),
     op_record("mod", 2, default_dtypes, all_shapes, jtu.rand_nonzero, []),
     op_record("modf", 1, float_dtypes, all_shapes, jtu.rand_default, []),
-    op_record("modf", 1, int_dtypes + unsigned_dtypes, all_shapes,
+    op_record("modf", 1, int_dtypes + unsigned_dtypes_no64, all_shapes,
               jtu.rand_default, [], check_dtypes=False),
     op_record("rint", 1, inexact_dtypes, all_shapes, jtu.rand_some_inf_and_nan,
               []),
-    op_record("rint", 1, int_dtypes + unsigned_dtypes, all_shapes,
+    op_record("rint", 1, int_dtypes + unsigned_dtypes_no64, all_shapes,
               jtu.rand_default, [], check_dtypes=False),
-    op_record("sign", 1, number_dtypes + unsigned_dtypes,
+    op_record("sign", 1, number_dtypes + unsigned_dtypes_no64,
```

Comment:
This also seems surprising. In classic NumPy, numpy.sign of a `uint64` yields a `uint64`.